umbra/umbra.h

```diff
@@ -389,6 +389,9 @@ DR_EXPORT
  * \note: \p num_scratch_regs must not be smaller than the value returned from
  * umbra_num_scratch_regs_for_translation, otherwise error code
  * DRMF_ERROR_NOT_ENOUGH_REGS is returned.
+ *
+ * \note: This method destroys aflags. Be sure to save and restore aflags before
+ * and after this method is called, i.e. with \p drreg_reserve_aflags.
```

Comment:
s/i.e./e.g.,/

Also if you put () after the function name doxygen will auto-link it (though prob won't work from DrM to DR since the docs are built separately)